Docker-Swarm-deployment/single-node/elk/README.md

```diff
@@ -1,84 +1,94 @@
-# Introduction
-Docker swarm stack for Elasticsearch-logstash-Kibana (ELK) Deployment.
-
-## ELK Installation
-## Create secret files
-1. Make a copy of sample secrets directory.
-```console
- cp -r example-secrets/secrets .
+# Install
+## Required secrets
+```sh
+secrets
+|-- passwords
+|   |-- elasticsearch-cat-password
+|   |-- elasticsearch-fs-password 
+|   |-- elasticsearch-rs-password
+|   |-- elasticsearch-su-password
+|   |-- kibana-admin-password
+|   |-- kibana-admin-username
+|   |-- kibana-system-password
+|   |-- logstash-internal-password
+|   |-- logstash-rabbitmq-password
+|   |-- logstash-rabbitmq-username
+|   `-- logstash-system-password
+`-- pki
+    |-- s3-access-key
+    `-- s3-secret-key
 ```
-2. To generate the passwords:
+Please see the example-secrets directory to get more idea, can use the 'secrets' in that directory by copying into root database directory  for demo or local testing purpose only! The kibana is tls secured through centralised nginx.
+
+## Assign node labels
 
-```console
-./create-secrets.sh
+```sh
+docker node update --label-add database_node=true <node_name>
 ```
```

Comment:
Why is the secret creation part removed?

Reply:
It was due to rebase issue, its fixed in next push.